include/MVC/Controller/SugarController.php

```diff
@@ -1110,7 +1110,7 @@ protected function remapAction()
     public function action_sendConfirmOptInEmail() {
         global $app_strings;
 
-        if ($this->bean instanceof Company || $this->bean instanceof Person) {
+        if (!in_array($this->bean->module_name, array('Accounts', 'Contacts', 'Leads', 'Prospects'))) {
```

Comment:
on second thought why does the check on Person and Company not work, ideally it would any Company or Person module excluding Users and including any custom modules of these types

Reply:
yeah, it was my first thought also, but for some reason it did not works. seems `instanceof` did not check for parent classes some reason and it shows up the error message when I was trying to send email.